multibody/contact_toolkit.h

```diff
@@ -12,17 +12,24 @@ namespace multibody {
 /*
  * ContactInfo structure that holds all the required contact information.
  * xA: Positions on Body A (The robot) expressed in the frame of the robot.
- * xB: Positions on Body B (Assumed to be the ground) expressed in the ground
  * frame (world frame).
- * idxA: Body indices of the corresponding positions in the RigidBodyTree.
+ * frameA: Frame of the corresponding positions in the MultibodyPlant.
```

Comment:
xA and frameA are not very informative variable names. How about we rename xA to contact_point, and frameA to contact_frame?